DEFCON Z/Assets/Tests/PlayMode Tests/MainMenuTest.cs

```diff
@@ -106,33 +149,27 @@ public void MainMenu_Should_Have_DifferentState_Of_MenuPanels_And_Settings()
         /// <param name="expectedMenuPanelState">Expected Menu Panel state after toggling.</param>
         /// <param name="expectedSettingsPanelState">Expected Settings Panel state after toggling.</param>
         /// <param name="expectedSettingsState">Expected Main Menu settings state after toggling.</param>
-        [TestCase(false, false, true, true)]
-        [TestCase(true, true, false, false)]
-        public void MainMenu_Should_ToggleSettings(bool initialSettingsState, bool expectedMenuPanelState, bool expectedSettingsPanelState, bool expectedSettingsState)
+        [TestCase(false, true, false, true)]
```

Comment:
Shouldn't this test just test whether the state of either of the menu is toggled? For e.g the starting state for Menu and Settings Menu is set to `false` and when toggled becomes `true`? Since you set the active state based on the given test case?

I recommend renaming the variable beforehand because `settings` boolean variable is no longer available in this patch.